python/lsst/ts/GenericCamera/driver/genericcamera.py

```diff
@@ -19,11 +19,32 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+__all__ = [
+    "GenericCamera",
+    "DATE_FORMAT",
```

Comment:
I am not sure `DATE_FORMAT` and `DATETIME_FORMAT` belong here. Maybe create a util module to host them?

Reply:
Done